libs/configuration-variables-resolver/src/VariablesRenderer/RegexRenderer.php

```diff
@@ -29,9 +29,13 @@ public function renderVariables(array $configuration, string $namePrefix, array
             // %s[a-zA-Z0-9_-]+ - match variable name (with prefix if supplied)
             // \s*}} - match optional whitespaces and closing braces
             sprintf('/(?<!{){{\s*(%s([a-zA-Z0-9_-]+))\s*}}/', preg_quote($namePrefix, '/')),
-            function (array $match) use ($values, &$missingVariables, &$replacedVariables): string {
+            function (array $match) use (&$values, &$missingVariables, &$replacedVariables): string {
                 [$placeholder, $prefixedName, $localName] = $match;
 
+                if (is_callable($values)) {
+                    $values = $values();
```

Comment:
proisím nějak inteligentně pojmenovat

Reply:
a ideálně zrušit array|callable a nechat tam jen callable

Reply:
array vyhozeno